packages/assets/src/utils/url/isAbsoluteUrl.ts

```diff
@@ -1,18 +1,23 @@
+import { settings } from '@pixi/settings';
+
 /**
  * Used to check whether the given URL is an absolute URL or not.
  * An absolute URL is defined as a URL that contains the complete details needed to locate a file
  *
- * Taken directly from here: https://github.com/sindresorhus/is-absolute-url/blob/master/index.js
+ * Adapted from here: https://github.com/sindresorhus/is-absolute-url/blob/master/index.js
  *
  * returns true if the URL is absolute, false if relative.
  * @param url - The URL to test
  */
 export function isAbsoluteUrl(url: string): boolean
 {
-    // Don't match Windows paths `c:\`
-    if ((/^[a-zA-Z]:\\/).test(url))
+    // If running in node then an absolute URL will start will either a `/` on unix or disk drive on windows
+    if (settings.ADAPTER.getNavigator().userAgent === 'node')
     {
-        return false;
+        if ((/^[a-zA-Z][a-zA-Z\d+\-.]*:/).test(url)) return true;
+        if (process.platform === 'win32') return (/^[a-zA-Z]:(\\|\/)/).test(url);
+
+        return url.startsWith('/');
```

Comment:
This seems like not the right place for this code. Part of the rationale of having an adapter is that is can externalize platform specifics (i.e., we should not have `process.platform` references in assets).

 An alternative approach:

```js
return settings.ADAPTER?.isAbsoluteUrl(url) ?? (/^[a-zA-Z][a-zA-Z\d+\-.]*:/).test(url);
```